test/unit/wizard-dialog.test.ts

```diff
@@ -258,4 +262,371 @@ describe('wizard-dialog', () => {
       after(() => localStorage.removeItem('mode'));
     });
   });
+
+  describe('with content definition throught WizardInput s', () => {
```

Comment:
```suggestion
  describe('with content definition throught WizardInputs', () => {
```

I'm better with spelling than code.